docs/readthedocs/models/pv_model.md

```diff
@@ -0,0 +1,483 @@
+(pv_model)=
+
+## PV Model
+
+This page documents the functionality of the PV model available in SIMONA.
+
+The initial parts of the model are presented in the paper [Agent based approach to model photovoltaic feed-in in distribution network planning](https://ieeexplore.ieee.org/abstract/document/7038345). Since then several adaptions has been made that are documented as follows.
+
+The PV Model is part of the SIMONA Simulation framework and represented by an agent.
+
+### Parameters
+
+```{mermaid}
+graph
+    A[(Setup-database)] 
+    B([PV Agent])
+    C{{Simulation environment}}
+
+    A -- "α<SUB>E</SUB> γ<SUB>E</SUB> <br> η<SUB>conv</SUB> cos(φ)<br> lon lat <br>  albedo<br> NodeID S<SUB>N</SUB><br> " --> B
+    B -- "P<SUB>el</SUB>(t)<br> Q<SUB>el</SUB>(t)"--> C
+    C -- "t <br> E<SUB>Dir, H</SUB> <br> E<SUB>Dif,H</SUB>" --> B
+```
+
+### Implemented Behaviour
+
+```{mermaid}
+flowchart
+    A[Setup] 
+    --> B[Receive new time step] 
+    --> C[Receive weather data]
+    --> D[Calculation of sun position]
+    --> E[Calculation of direct radiation]
+    --> F[Calculation of diffuse radiation]
+    --> G[Calculation of reflected radiation]
+    --> H[Estimation of correction factors]
+    --> I[Calculation of output power P]
+    --> J[Calculation of reactive power Q]
+    --> K[Return p and Q to node agent] --> B
+```
+
+### Output visualization
+
+![](../images/models/pv_model/pv_it_test_wholeyear.png)
+
+### Calculations
+
+![](../images/models/pv_model/pvradiation.png)
```

Comment:
Are those figures of us? I'm a bit suspicious, that it's not the case... @johanneshiry?

Reply:
I think you're right. I removed them but contacted Johannes. If copyright will allow the usage I will reintegrate them.